Description: Link TPLs privately.
Author: Nico Schlömer <nico.schloemer@gmail.com>
Forwarded: not-needed

--- a/liblib/CMakeLists.txt
+++ b/liblib/CMakeLists.txt
@@ -121,7 +121,7 @@ IF(TLL_LIBS)
   LIST(REMOVE_DUPLICATES TLL_LIBS)
 ENDIF()
 
-TARGET_LINK_LIBRARIES(netcdf ${TLL_LIBS})
+TARGET_LINK_LIBRARIES(netcdf PRIVATE ${TLL_LIBS})
 
 SET(CMAKE_REQUIRED_LIBRARIES ${CMAKE_REQUIRED_LIBRARIES} ${TLL_LIBS})
 IF(MSVC)
